modules/node_modules/@ncigdc/utils/formatFileSize.js

```diff
@@ -1,31 +1,4 @@
 // @flow
+import filesize from 'filesize';
```

Comment:
why not delete this thin wrapper and call `filesize()` directly instead of `formatFileSize`. Or maybe use `/components/FileSize.js`? Could you filesize this as well https://github.com/NCI-GDC/portal-ui/blob/relay/modules/node_modules/%40ncigdc/components/File.js#L143 while you're at it